packages/ckeditor5-engine/src/conversion/downcasthelpers.js

```diff
@@ -357,6 +358,113 @@ export default class DowncastHelpers extends ConversionHelpers {
 	markerToHighlight( config ) {
 		return this.add( downcastMarkerToHighlight( config ) );
 	}
+
+	/**
+	 * Model marker converter for data downcast.
+	 *
+	 * This conversion creates a representation for model marker boundaries in the view:
+	 *
+	 * * if the marker boundary is at a position where text nodes are allowed, then a view element with specified tag name
+	 * and `name` attribute is added at that position,
+	 * * in other case, a specified attribute is set on an view element that is before/after marker boundary.
+	 *
+	 * Tag and attribute names and values are generated from the marker name:
+	 *
+	 * * templates for attributes are `data-group-start-before="name"`, `data-group-start-after="name"`,
```

Comment:
Maybe like that?

```suggestion
`data-[group]-start-before="[name]"`
```

It's hard to understand which parts of that template are variables.

Reply:
I am always afraid that somebody would think that `[]` are also the part of the name (`data-[comment]-start-before="[123:abc]"`) :D